syft/cataloging/license.go

```diff
@@ -14,10 +14,6 @@ const (
 )
 
 type LicenseConfig struct {
-	// IncludeUnknownLicenseContent controls whether the content of a license should be included in the SBOM when the license ID cannot be determined.
-	// Deprecated: use IncludeContent instead
-	IncludeUnknownLicenseContent bool `json:"-" yaml:"-" mapstructure:"-"`
```

Comment:
Removing this is technically a breaking change, should we leave this deprecated? It looks like this option has been [in the codebase for a little while](https://github.com/anchore/syft/blob/e584c9f416028051fbdc80fffda4e624aaed528d/syft/cataloging/license.go)

Reply:
we discovered it wasn't wired up, thus it wasn't actually usable. It is possible that someone was blindly using the struct, thus changing it would be a compile error, but seeing that doing so would have no effect. 

Given this, this is a pretty good time to remove it.

Reply:
@wagoodman mentioned this in an offline sync. Because it was NOT wired up I think we're ok to remove this since it will not break anyone's behavior